neofs/neofs_contract.go

```diff
@@ -338,9 +343,15 @@ func InnerRingUpdate(chequeID []byte, args [][]byte) bool {
 	irList := getInnerRingNodes(ctx, innerRingKey)
 	threshold := len(irList)/3*2 + 1
 
-	irKey := innerRingInvoker(irList)
-	if len(irKey) == 0 {
-		panic("innerRingUpdate: invoked by non inner ring node")
+	voteKey := innerRingInvoker(irList)
+	if len(voteKey) == 0 {
+		committee := getCommitteeKeys()
+		threshold = len(committee)/3*2 + 1
+
+		voteKey = committeeInvoker(committee)
```

Comment:
And this treats IR votes and committee votes equal, I'm not sure about that either.